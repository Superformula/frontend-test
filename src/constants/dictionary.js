export const DICTIONARY = Object.freeze({
  OPEN: 'open',
  OPEN_NOW: 'open now',
  CLOSED: 'closed',
  RESTAURANTS: 'Restaurants',
  ALL_RESTAURANTS: 'All Restaurants',
  MAIN_PAGE_DESCRIPTION:
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
  PRICE: 'Price',
  CATEGORIES: 'Categories',
  CLEAR_ALL: 'Clear all',
  LEARN_MORE: 'Learn more',
  LOAD_MORE: 'Load more',
  APPLY: 'Apply',
  SELECT_ONE: 'Select One',
  FILTER_BY: 'Filter by',
  TOGGLE: 'toggle',
});
