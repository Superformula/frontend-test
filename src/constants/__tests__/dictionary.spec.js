import { DICTIONARY } from '../dictionary';

describe('DICTIONARY', () => {
  it('should have proper values', () => {
    expect(DICTIONARY).toMatchSnapshot();
  });
});
