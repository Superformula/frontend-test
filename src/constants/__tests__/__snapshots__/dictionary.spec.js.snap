// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DICTIONARY should have proper values 1`] = `
Object {
  "ALL_RESTAURANTS": "All Restaurants",
  "APPLY": "Apply",
  "CATEGORIES": "Categories",
  "CLEAR_ALL": "Clear all",
  "CLOSED": "closed",
  "FILTER_BY": "Filter by",
  "LEARN_MORE": "Learn more",
  "LOAD_MORE": "Load more",
  "MAIN_PAGE_DESCRIPTION": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
  "OPEN": "open",
  "OPEN_NOW": "open now",
  "PRICE": "Price",
  "RESTAURANTS": "Restaurants",
  "SELECT_ONE": "Select One",
  "TOGGLE": "toggle",
}
`;
